bottom-bar/src/main/java/com/roughike/bottombar/BottomBarTab.java

```diff
@@ -92,6 +93,11 @@ void prepareLayout() {
         setGravity(Gravity.CENTER_HORIZONTAL);
         setLayoutParams(new LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT));
 
+        // set a selectable item background
+        final TypedValue typedValue = new TypedValue();
```

Comment:
@yombunker @MarcRubio 

See the MiscUtils.java, which has a [pretty similar method for resolving attributes](https://github.com/roughike/BottomBar/blob/master/bottom-bar/src/main/java/com/roughike/bottombar/MiscUtils.java#L38-L42).

The method is incorrectly named to ```getColor```, but should probably be ```getResourceAttribute```, and the **@ColorRes** annotation should be replaced with **@AttrRes** to get rid of the lint warning in the BottomBar class.

See if you could make that method reusable for this and resolving the primary color, so this wouldn't have to be done inline.